server/sonar-server/src/main/java/org/sonar/server/computation/issue/EffortAggregator.java

```diff
@@ -77,36 +82,38 @@ public void afterComponent(Component component) {
     Metric metric = metricRepository.getByKey(CoreMetrics.TECHNICAL_DEBT_KEY);
 
     // total value
-    measureRepository.add(component, metric, Measure.newMeasureBuilder().create(this.currentDebt.minutes));
+    measureRepository.add(component, metric, Measure.newMeasureBuilder().create(this.effort.maintainabilityEffort));
 
+    // TODO delete following lines when working on SONAR-7425
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)